packages/cli/lib/index.js

```diff
@@ -51,7 +51,7 @@ prog
 	)
 	.option('-c, --config', 'Path to custom CLI config', 'preact.config.js')
 	.option('--esm', 'Builds ES-2015 bundles for your code.', true)
-	.option('--brotli', 'Adds brotli redirects to the service worker.', false)
+	.option('--brotli', 'Builds brotli compressed bundles of javascript.', false)
```

Comment:
For now, should we not remove this option?

Reply:
I dont think so,  this still generates the bundles.  which can be used by people for their custom server setups

Reply:
Yep true, forgot about that part.